### PR TITLE
PL: CSF Intro post-workshop survey updates

### PR DESCRIPTION
**Pull request by breville**
As well as implementing the latest content, this uses SurveyJS' dynamic panel feature to show repeated items for each facilitator.

We were originally going to have separate surveys for each facilitator, after the main survey, which is how the legacy JotForm system does it, but learned that it can be better to ask demographic information after asking everything else to avoid influencing how the submitter thinks about those answers, and so we're going to try for this single-survey approach instead.

foorm-2

### CODE REVIEW

File: dashboard/config/foorm/forms/surveys/pd/workshop_csf_intro_post.0.json [hunk 1]
```diff
@@ -7,115 +7,80 @@
...
-          "choices": [
+          "type": "matrix",
+          "name": "overall_success",
+          "title": "Overall",
+          "description": "How much do you agree or disagree with the following statements about the workshop overall?",
+          "columns": [
             {
-              "value": "yes",
-              "text": "Yes"
+              "value": "strongly_disagree",
```
Comment (molly-moen) [inline dashboard/config/foorm/forms/surveys/pd/workshop_csf_intro_post.0.json]: for matrices/other questions we may want to roll up, can we move to using numeric values? That way we can roll them up

Comment (breville) [reply dashboard/config/foorm/forms/surveys/pd/workshop_csf_intro_post.0.json]: Good call.

**Review by molly-moen — APPROVED** [review]
👍

**Review by molly-moen — APPROVED** [review]
Nice!

**Pull request merged**